docs/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -0,0 +1,300 @@
+        - `<image-url>` is the URL of the image container.
+        For example, `gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/event_display`
+   </br></br>
+    2. Apply the YAML file by running:
+        ```bash
+        kubectl apply --filename service.yaml
+      ```
+
+
+
+
+## Create a subject
+
+The subject must be a PodSpecable resource.
+You can use any PodSpecable resource in your cluster, for example:
+
+- `Deployment`
+- `Job`
+- `DaemonSet`
+- `StatefulSet`
+- `Service.serving.knative.dev`
+
+If you do not have an existing PodSpecable subject that you want to use, you can
+use the following sample to create a `CronJob` object as the subject.
+The following cron job makes a single cloud event that targets `K_SINK` and adds
+any extra overrides given by `CE_OVERRIDES`.
+
+1. Copy the sample YAML below into a `cronjob.yaml` file:
+
+    ```yaml
+    apiVersion: batch/v1beta1
+    kind: CronJob
+    metadata:
+      name: heartbeat-cron
+    spec:
+      # Run every minute
+      schedule: "*/1 * * * *"
+      jobTemplate:
+        metadata:
+          labels:
+            app: heartbeat-cron
+        spec:
+          template:
+            spec:
+              restartPolicy: Never
+              containers:
+                - name: single-heartbeat
+                  image: gcr.io/knative-releases/knative.dev/eventing-contrib/cmd/heartbeats
+                  args:
+                  - --period=1
+                  env:
+                    - name: ONE_SHOT
+                      value: "true"
+                    - name: POD_NAME
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.name
+                    - name: POD_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.namespace
+    ```
+
+2. Apply the YAML file by running:
+
+    ```bash
+    kubectl apply --filename cronjob.yaml
+    ```
+
+
+## Create a SinkBinding object
+
+Create a `SinkBinding` object that directs events from your subject to the event sink.
+
+=== "kn"
+
+    Create a `SinkBinding` object by running:
+
+    ```bash
+    kn source binding create <name> \
+      --namespace <namespace> \
+      --subject "<subject>" \
+      --sink <event-sink> \
+      --ce-override "<cloudevent-overrides>"
+    ```
+    Where:
+    - `<name>` is the name of the SinkBinding object you want to create.
+    - `<namespace>` is the namespace you created for your SinkBinding to use.
+    - `<subject>` is the subject to connect. Examples:
+      - `Job:batch/v1:app=heartbeat-cron` matches all jobs in namespace with label `app=heartbeat-cron`.
+      - `Deployment:apps/v1:myapp` matches a deployment called `myapp` in the namespace.
+      - `Service:serving.knative.dev/v1:hello` matches the service called `hello`.
+    - `<event-sink>` is the event sink to connect. For example `http://event-display.svc.cluster.local`.
```

Comment:
same comments as before - need consistent formatting, and use "sink" not "event sink" IMO